README.md

```diff
@@ -40,15 +41,16 @@ services:
       DATABASE_URL: "postgresql://postgres:postgres@db:5432/postgres"
     depends_on:
       db:
-        condition: service_started
+        condition: service_healthy
 
   agent:
     image: haedlessdev/portnote-agent:latest
+    restart: unless-stopped
     environment:
       DATABASE_URL: "postgresql://postgres:postgres@db:5432/postgres"
     depends_on:
       db:
-        condition: service_started
+        condition: service_healthy
 
   db:
     image: postgres:17
```

Comment:
Maybe since you added the `restart: unless-stopped` in other containers it would be good to add to postgres as well?

Reply:
The postgres container has `restart: always` currently but I suppose I'll change it to `unless-stopped` for consistency.